package.json

```diff
@@ -70,5 +71,10 @@
   "packageManager": "pnpm@7.5.0",
   "engines": {
     "pnpm": ">=7"
+  },
+  "pnpm": {
+    "overrides": {
+      "esbuild": "0.15.13"
```

Comment:
workaround for esbuild issue. was supposed to be fixed by [`esbuild@0.15.14`](https://github.com/evanw/esbuild/releases/tag/v0.15.14), but that seems to have broken things.